README.md

```diff
@@ -16,7 +16,7 @@ The background jobs packages are easy to install via NuGet.
     PM > Install-Package Arcus.BackgroundJobs.CloudEvents
     ```
 ## Security
-  - Azure Active Directory background jobs notifies event subscriptions on expired client secrets in an Azure Active Directory.
+  - Azure Active Directory background jobs [notifies event subscriptions on expired client secrets in an Azure Active Directory](https://background-jobs.arcus-azure.net/Features/azureactivedirectory/client-secret-expiration-job/).
```

Comment:
So, this link is not active yet, but should be part of the release.
Hmm, we could also opt to wait for this to update and do it in reverse: first update the feature docs for v0.4 and then do the actual NuGet release trigger. That way, we are sure that the link provided here in the README is still correct.

Reply:
What is the amount of time between the nuget release & the update of the docs ?

Reply:
Whatever we want it to be. 😄 We can already make a PR for the docs before we start the release.

Reply:
Ok, just checked, it has the correct URL here, so we're good.